File: src/app/shared/order.ts
import { Data } from '@angular/router';
// tslint:disable-next-line:import-spacing
import {Customer} from  './customer';

export interface Order{
    orders: any;
    id: number;
    customer: Customer;
    total: number;
    placed: Data;
    fulfilled: Data;
    status: string;
}

File: Advantage.API/Advantage.API/Models/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Advantage.API.Models
{
  public class Order
  {

    public int Id { get; set; }
    public Customer customer { get; set; }
    public int Total { get; set; }
    public DateTime placed { get; set; }
    public DateTime? fulfilled { get; set; }

    public string status { get; set; }


  }
}
